dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/BooleanLiteralNode.java

```diff
@@ -48,10 +44,4 @@ public boolean equals(@Nullable Object obj) {
         // super method compares values
         return super.equals(obj);
     }
-
-    @Override
-    @SideEffectFree
-    public Collection<Node> getOperands() {
```

Comment:
Should `getOperands()` be final in `ValueLiteralNode` to avoid this from re-occurring?